build.gradle

```diff
@@ -1,5 +1,17 @@
 defaultTasks ':core:jar'
 
+// Top-level build file where you can add configuration options common to
+// all sub-projects/modules.
+buildscript {
```

Comment:
This section assures that the same third party version is used along all the build